app/javascript/components/close_contact/CloseContact.js

```diff
@@ -183,41 +220,50 @@ class CloseContact extends React.Component {
     return (
       <React.Fragment>
         {!this.props.close_contact.id && (
-          <Button onClick={this.toggleModal}>
-            <i className="fas fa-plus"></i> Add New Close Contact
-          </Button>
+          <div className="pl-2">
+            <Button onClick={this.toggleModal}>
```

Comment:
it might have been like this already before, but the state does not reset after clicking cancel on the modal so when the modal is opened again, the old input data is still there

Reply:
This was the behavior before, but I do agree that it should be nulled! Going to change that in the latest commit.